onnxruntime/test/optimizer/graph_transform_utils_test.cc

```diff
@@ -55,7 +55,7 @@ TEST(GraphTransformerUtilsTests, TestGenerateGraphTransformers) {
 TEST(GraphTransformerUtilsTests, TestGenerateGraphTransformers_CustomList) {
   // custom list of rules and transformers
   std::string l1_rule1 = "EliminateIdentity";
-  std::string l1_rule2 = "ConstantFolding";
+  std::string l1_rule2 = "EliminateSlice";
```

Comment:
you can keep constant folding here and check custom list works with rule and transformer in same level. Same test you mentioned in my last PR. Back then we did not have such a case so I could not add it and we said you can add it once you make this change.

Reply:
Sure, will do that.